angular.json

```diff
@@ -3,34 +3,29 @@
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
-    "technical-assistant": {
+    "navbar": {
       "projectType": "application",
-      "schematics": {
-        "@schematics/angular:component": {
-          "style": "scss"
-        }
-      },
+      "schematics": {},
       "root": "",
       "sourceRoot": "src",
       "prefix": "app",
       "architect": {
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
-            "outputPath": "dist/technical-assistant",
+            "outputPath": "dist/navbar",
             "index": "src/index.html",
             "main": "src/main.ts",
             "polyfills": [
               "zone.js"
             ],
             "tsConfig": "tsconfig.app.json",
-            "inlineStyleLanguage": "scss",
```

Comment:
Why was SCSS support removed from the project? Add SCSS support again and the whole application should use SCSS instead of CSS.